powerline/segments/i3wm.py

```diff
@@ -19,8 +22,14 @@ def calcgrp(w):
 	return group
 
 
+def format_name(name, strip=False):
+	if strip:
```

Comment:
If strip is integer it should still exhibit the same behaviour. Also for checking type it is recommended to use `isinstance`.